### PR TITLE
added style files support for 'source.vue'

### PR DESCRIPTION
**Pull request by AlexQuidditch**
Hi there!

I added support for style files inside <style></style> tag. There wasn't to able for autocomplete paths before.

It's my first PR :D

### CODE REVIEW

File: lib/config/default-scopes.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ export default [
       'import\\s+.*?from\\s+[\'"]', // import foo from './foo'
       'import\s+[\'"]', // import './foo'
       'require\\([\'"]', // require('./foo')
-      'define\\(\\[?[\'"]' // define(['./foo']) or define('./foo')
+      'define\\(\\[?[\'"]', // define(['./foo']) or define('./foo')
+      '@import[\\(|\\s+]?[\'"]', // @import 'foo' or @import('foo')
     ],
-    extensions: ['js', 'jsx', 'vue', 'ts', 'tsx', 'coffee'],
+    extensions: ['js', 'jsx', 'vue', 'ts', 'tsx', 'coffee','sass', 'scss', 'css', 'less'],
```
Comment (Alexance) [inline lib/config/default-scopes.js]: Why you didn't add extensions for Stylus files? Default boilerplate from vue-cli supports it right out-of-box.

Comment (AlexQuidditch) [reply lib/config/default-scopes.js]: Sorry, I forgot about it :(

File: lib/config/default-scopes.js [hunk 2]
```diff
@@ -7,9 +7,10 @@ export default [
       'import\\s+.*?from\\s+[\'"]', // import foo from './foo'
       'import\s+[\'"]', // import './foo'
       'require\\([\'"]', // require('./foo')
-      'define\\(\\[?[\'"]' // define(['./foo']) or define('./foo')
+      'define\\(\\[?[\'"]', // define(['./foo']) or define('./foo')
+      '@import[\\(|\\s+]?[\'"]', // @import 'foo' or @import('foo')
```
[the remark this answers is not in the document]
Comment (saschagehlich) [reply lib/config/default-scopes.js]: Yeah, I'll move it to a seperate scope. Separation of concerns is probably a good idea here.

**Comment by saschagehlich** [comment]
I incorporated this change in 2.11.0. Thanks!

**Pull request closed without merging**